frontend/public/components/project-overview.jsx

```diff
@@ -0,0 +1,341 @@
+import * as React from 'react';
+import * as _ from 'lodash-es';
+import * as classNames from 'classnames';
+import { Button, EmptyState, Icon, Toolbar } from 'patternfly-react';
+import { Helmet } from 'react-helmet';
+import { StartGuide } from './start-guide';
+import { FLAGS, connectToFlags, flagPending } from '../features';
+import { Firehose, NavTitle, ResourceName, SectionHeading } from './utils';
+import { DeploymentConfigOverviewList } from './deployment-config';
+
+const LIST_VIEW = 'list';
+const GRAPH_VIEW = 'graph';
+
+const ProjectOverviewToolbar = ({activeView, setView}) => {
+  return <Toolbar>
+    <Toolbar.RightContent>
+      <Toolbar.ViewSelector>
+        <Button
+          bsStyle="link"
+          className={classNames({ active: activeView === LIST_VIEW })}
+          onClick={() => setView(LIST_VIEW)}
+          title="Topology List" >
+          <Icon type="fa" name="th-list" />
+        </Button>
+        <Button
+          bsStyle="link"
+          className={classNames({ active: activeView === GRAPH_VIEW })}
+          onClick={() => setView(GRAPH_VIEW)}
+          title="Topology Graph" >
+          <Icon type="pf" name="topology" />
+        </Button>
+      </Toolbar.ViewSelector>
+    </Toolbar.RightContent>
+  </Toolbar>;
+};
+
+const DeploymentConfigOverview = ({obj}) => {
```

Comment:
I'm inclined to split this out early into separate component files, as well as see how generic they can be (perhaps smaller pieces will allow for generic lists with some nested child components for customization, perhaps imported from our existing `deployment-config.tsx`, etc files, not sure).  

Thinking about structure early for deduplication sake.

Reply:
Ah, I see you are doing some of that already, nice.

Reply:
Yes, plan on doing similar breaking down for other resources that will be displayed in this view, but just haven't gotten there yet. I wanted to put up a PR so that others could see what I am working on and start getting feedback early, but there's still work to be done.